rules-topics/01_introduction.md

```diff
@@ -90,30 +90,27 @@ Status that require Identify:
 |Spirit Bottle|
 
 ### Floating Reset
-Some skills or abilities are listed as "Daily".  These skills and abilities are renewed between 4:00 PM and 6:00 PM.  If you are on a module between that time frame, you must attempt the reset immediately after the mod.  Daily skills and abilities require 10 minutes of concentration/rest to refresh.
+Some skills or abilities are listed as "Daily".  These skills and abilities are renewed between 4:00 PM and 6:00 PM.  If a player is on a module (an event during game run by plot) between that time frame, they must attempt the reset immediately after it ends.  Daily skills and abilities require 10 minutes of concentration/rest to refresh.
```

Comment:
Maybe make a mention of Rest here?

Reply:
Added a small blurb for Concentration & Rest.